common/dynamicconfig/registry.go

```diff
@@ -17,6 +17,14 @@ var (
 	globalRegistry registry
 )
 
+func GetDefaultValueForKey(k Key) ConstrainedValue {
```

Comment:
as above, this just doesn't make any sense for settings with constrained default values, there is no single default.